csharp/README.md

```diff
@@ -47,34 +42,30 @@ To build all Ice assemblies and the associated test suite, run:
 msbuild msbuild\ice.proj
 ```
 
-Upon completion, the Ice assemblies for .NET 6.0, the .NET Framework 4.5 and .NET Standard
-2.0 are placed in the `lib\net6.0`, `lib\net45` and `lib\netstandard2.0` folders respectively.
-
-> Note: the assemblies for .NET 6.0 are created only when you build with Visual Studio 2022
-> or greater.
-
-> Note: the assemblies for .NET Standard 2.0 are created only when you build with Visual Studio
-> 2017 or greater.
+Upon completion, the Ice assemblies for the .NET Framework 4.5 and .NET Standard 2.0 are placed
+in the `lib\net45` and `lib\netstandard2.0` folders respectively.
 
 You can skip the build of the test suite with the `BuildDist` target:
 ```
 msbuild msbuild\ice.proj /t:BuildDist
 ```
 
-The `Net6Build`, `Net6BuildDist`, `Net45Build`, `Net45BuildDist`, `NetStandardBuild` and
-`NetStandardBuildDist` targets allow you to build assemblies only for .NET 6.0, the .NET
-Framework 4.5 or .NET Standard 2.0, with or without the test suite.
+The `Net45Build`, `Net45BuildDist`, `NetStandardBuild` and `NetStandardBuildDist` targets allow
+you to build assemblies only for the .NET Framework 4.5 or .NET Standard 2.0, with or without
+the test suite.
+
+> Note: Visual Studio 2022 version or higher is required for .NET Standard 2.0 builds.
 
-The .NET Standard build of iceboxnet and test applications target `netcoreapp3.1` when using
-Visual Studio 2019 and `netcoreapp2.1` when using Visual Studio 2017. You can change
+The .NET Standard build of iceboxnet and test applications target `net6.0` You can change
```

Comment:
Is `net6.0` correct here?

Reply:
Yes, the libraries target .Net Standard 2.0, but executables need to target a concrete framework, by default all executables in our build target .NET 6.0 which supports the .Net Standard 2.0.